.travis.yml

```diff
@@ -110,6 +115,20 @@ jobs:
       php: "7.3"
       env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
 
+    - stage: coverage
+      name: PHP and JavaScript unit tests (7.3, WordPress trunk, with code coverage)
+      if: branch = develop
+      php: "7.3"
+      env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
+      before_script:
+        - echo "Running unit tests with code coverage..."
+      script:
+        - npm run build:js
+        - source "$DEV_LIB_PATH/travis.script.sh"
+        - bash <(curl -s https://codecov.io/bash) -cF php
+        - npm run test:js
+        - bash <(curl -s https://codecov.io/bash) -cF javascript
```

Comment:
If this doesn't work, I'd suggest trying https://www.npmjs.com/package/codecov.

See [here](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/codecov-upload.js#L27-L47) for how it's used by `ampproject/amphtml`.